services/web/server/src/simcore_service_webserver/director_v2_handlers.py

```diff
@@ -26,42 +31,63 @@
 @permission_required("project.read")
 @log_decorator(logger=log)
 async def start_pipeline(request: web.Request) -> web.Response:
-    director2_settings: Directorv2Settings = get_settings(request.app)
+    client = DirectorV2ApiClient(request.app)
+
+    run_policy = get_run_policy(request.app)
+    assert run_policy  # nosec
+
+    user_id = UserID(request[RQT_USERID_KEY])
+    project_id = UUID(request.match_info["project_id"])
 
-    user_id = request[RQT_USERID_KEY]
-    project_id = request.match_info.get("project_id", None)
     subgraph: Set[str] = set()
-    force_restart = False
+    force_restart: bool = False
     cluster_id: NonNegativeInt = 0
+
     if request.can_read_body:
         body = await request.json()
-        subgraph = body.get("subgraph")
-        force_restart = body.get("force_restart")
+        subgraph = body.get("subgraph", [])
+        force_restart = bool(body.get("force_restart"))
         cluster_id = body.get("cluster_id")
 
-    backend_url = URL(f"{director2_settings.endpoint}/computations")
-    log.debug("Redirecting '%s' -> '%s'", request.url, backend_url)
-    body = {
-        "user_id": user_id,
-        "project_id": project_id,
+    options = {
         "start_pipeline": True,
         "subgraph": list(subgraph),  # sets are not natively json serializable
         "force_restart": force_restart,
         "cluster_id": cluster_id,
     }
 
-    # request to director-v2
     try:
-        computation_task_out = await _request_director_v2(
-            request.app, "POST", backend_url, expected_status=web.HTTPCreated, data=body
+        project_ids: List[ProjectID]
+        project_vc_commits: List[CommitID]
+
+        (
+            project_ids,
+            project_vc_commits,
+        ) = await run_policy.get_or_create_runnable_projects(request, project_id)
+        log.debug("Project %s will start %d variants", project_id, len(project_ids))
+
+        assert project_ids  # nosec
+        assert (  # nosec
+            len(project_ids) == len(project_vc_commits) if project_vc_commits else True
+        )
+
+        _started_pipelines_ids: Tuple[str] = await asyncio.gather(
```

Comment:
use logged_gather here?

Reply:
for the moment, i would like to avoid adding logs to asyn tasks until we have the "blocking effect of logs" fully clarified

Reply:
To be honest, from what I see asyncio itself uses logging as its logging as its own logging facilities. Don't you think that until proved otherwise we should keep in using it? 
Most of the libraries are using it and it seems we can just change some handler to make it fully async later:
see [this](https://www.zopatista.com/python/2019/05/11/asyncio-logging/)